### ISSUE TITLE
Rename repository to match our agency’s name

### ISSUE THREAD

**Issue by klepas** [AusDTO/dto-design-guide#22]
Couldn’t find an issue for this and figured we should do this to this repo before we make it public to anyone within the agency so that we avoid the URL problems we’re gonna have with the UI-Kit’s repo rename.

**Comment by gazhands** [comment]
Chat to cloud

**Comment by dominikwilkowski** [comment]
Shall we wait with this until the GitHub org has also been changed as we otherwise have to go through this pain twice?

**Comment by klepas** [comment]
> Shall we wait with this until the GitHub org has also been changed as we otherwise have to go through this pain twice?

Yes. We should actually raise this in #tech or the next guild meeting.

**Comment by klepas** [comment]
Had a chat to Catherine from cloud: cut-over date for when the AusDTO org name will change is anticipated to be `Jan 16 2017` — we should rename our repos on or after this date methinks @dominikwilkowski ? :)
